src/main/java/org/folio/clients/CirculationRequestClient.java

```diff
@@ -159,11 +164,22 @@ private JsonParser getJsonParser(Promise<Map<String, Long>> promise) {
         }
         Map<String, Long> requestMap = requestsResp.getRequests()
             .stream()
+            .filter(this::removeClosedCancelledRequests)
             .collect(groupingBy(Request::getItemId, counting()));
         promise.complete(requestMap);
       });
   }
 
+  private boolean removeClosedCancelledRequests(Request itemRequest) {
+    try {
+      EnumSet<Request.Status> openStatuses = EnumSet.of(
+          OPEN_NOT_YET_FILLED, OPEN_AWAITING_PICKUP, OPEN_IN_TRANSIT, OPEN_AWAITING_DELIVERY);
+      return openStatuses.contains(itemRequest.getStatus());
+    } catch (Exception e) {
+      return false;
+    }
+  }
```

Comment:
Might be more readable to name this method `isOpenStatus` since it's not touching the closed statuses.

Another possibility if you want to use the closed statuses (maybe better):
```
private boolean isClosedStatus(Request itemRequest) {
  EnumSet<Request.Status> openStatuses = EnumSet.of(CLOSED_CANCELLED, CLOSED_FILLED);
  return openStatuses.contains(itemRequest.getStatus());
}
```

And then use `Predicate.not` like this `.filter(Predicate.not(this::isClosedStatus))`.

Also I wouldn't wrap this in a try catch since it doesn't seem like an exception would be thrown. And even if there was the remote chance, this would mask that. IOW we'd want to know about it.

Reply:
Good catch. Fixed them